CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # mdn-bcd-collector Changelog
 
+## v6.2.11
+
+### Test Changes
+
+#### Added
+
+- api.Selection.getComposedRange
+- api.Window.beforecopy_event
+- api.Window.beforecut_event
+- api.Window.beforepaste_event
+- api.Window.copy_event
+- api.Window.cut_event
+- api.Window.paste_event
+
+#### Removed
+
+- api.Document.rejectionhandled_event
+- api.Document.unhandledrejection_event
+- api.HTMLElement.rejectionhandled_event
+- api.HTMLElement.unhandledrejection_event
+- api.MathMLElement.rejectionhandled_event
+- api.MathMLElement.unhandledrejection_event
+- api.PressureObserver.requestPermission
```

Comment:
Not in https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/compute_pressure/pressure_observer.idl;drc=1ac74b47029c948c021372a7ffe2043892060acf